tests/test_ssm/test_ssm_boto3.py

```diff
@@ -227,76 +227,73 @@ def test_get_parameters_by_path():
     )
 
 
+@pytest.mark.parametrize("name", ["test", "my-cool-parameter"])
 @mock_ssm
-def test_put_parameter():
+def test_put_parameter(name):
     client = boto3.client("ssm", region_name="us-east-1")
 
     response = client.put_parameter(
-        Name="test", Description="A test parameter", Value="value", Type="String"
+        Name=name, Description="A test parameter", Value="value", Type="String"
     )
 
     response["Version"].should.equal(1)
 
-    response = client.get_parameters(Names=["test"], WithDecryption=False)
+    response = client.get_parameters(Names=[name], WithDecryption=False)
 
     len(response["Parameters"]).should.equal(1)
-    response["Parameters"][0]["Name"].should.equal("test")
+    response["Parameters"][0]["Name"].should.equal(name)
     response["Parameters"][0]["Value"].should.equal("value")
     response["Parameters"][0]["Type"].should.equal("String")
     response["Parameters"][0]["Version"].should.equal(1)
     response["Parameters"][0]["LastModifiedDate"].should.be.a(datetime.datetime)
     response["Parameters"][0]["ARN"].should.equal(
-        "arn:aws:ssm:us-east-1:1234567890:parameter/test"
+        f"arn:aws:ssm:us-east-1:1234567890:parameter/{name}"
```

Comment:
We still support Py2, so f-strings will not work unfortunately.
We usually use "". format

Reply:
Thanks for the feedback. I took care of this 🎉  Since I pushed up the change and not Kyle, it's not running the build.